test/test_webhooks.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'test_helper'
+
+# Internal: The test for Plaid::Webhooks.
+class PlaidWebhooksTest < PlaidTest
+  # rubocop:disable Metrics/AbcSize
+  def test_get_verification_key
+    key_id = '6c5516e1-92dc-479e-a8ff-5a51992e0001'
+
+    webhook_verification_key_response =
+      client.webhooks.get_verification_key(key_id)
+    refute_empty(webhook_verification_key_response.key)
```

Comment:
I'm assuming these tests will always pass and none of these fields are optional? Otherwise tests will flake.

Reply:
correct, none of these fields will ever be absent.